app/Http/Controllers/Users/UsersController.php

```diff
@@ -611,6 +612,18 @@ public function printInventory($id)
             ->with('show_user', $show_user)
             ->with('settings', Setting::getSettings());
     }
+    public function emailAssetList($id)
+    {
+        $this->authorize('view', User::class);
+        $user = User::where('id', $id)->withTrashed()->first();
```

Comment:
Do we want `withTrashedI()` here? Is there a use case where we'd be sending an email to a deleted user?

Also we should be checking that the user actually exists before trying to action on it.